components/nifosg/particle.cpp

```diff
@@ -159,8 +183,11 @@ void ParticleColorAffector::operate(osgParticle::Particle* particle, double /* d
 {
     float time = static_cast<float>(particle->getAge()/particle->getLifeTime());
     osg::Vec4f color = mData.interpKey(time);
+    float alpha = color.a();
+    color.a() = 1.0f;
 
     particle->setColorRange(osgParticle::rangev4(color, color));
+    particle->setAlphaRange(osgParticle::rangef(alpha, alpha));
```

Comment:
AFAIK ignoring alpha component of the used color isn't a part of upstream osgParticle (and somehow I can't find your PR where you tried to change that). While the way this is done won't break anything these few changes above don't seem to be necessary at the moment and aren't relevant to the issue.

Reply:
this is mandatory because  color and alpha are separated in osg, i don't ignore alpha just put it in the right channel